doc/mgr/orchestrator.rst

```diff
@@ -550,6 +550,25 @@ Or with hosts:
         - host2
         - host3
 
+Updating Service Specifications
+===============================
+
+The Ceph Orchestrator maintains a declarative state of each
+service in a ``ServiceSpec``. For certain operations, like updating
+the RGW HTTP port, we need to update the existing
+specification.
+
+First, get the current ``ServiceSpec``::
+
+    ceph orch ls --service_name=<service-name> --export > myservice.yaml
+
+Now, update the yaml file (like changing the port)::
```

Comment:
```suggestion
2. Update the yaml file (like changing the port)::

Reply:
What does "(like changing the port)" mean?

Reply:
like changing the `spec.rgw_frontend_port` property of the spec.

Reply:
I think maybe make that clear and expand that if you're  going to include "like changing the port" or link to another section that details these instructions? It looks a little out of place.